app/scripts/lib/rpc-method-middleware/handlers/wallet-createSession/handler.ts

```diff
@@ -162,6 +162,7 @@ async function walletCreateSessionHandler(
       requiredScopes: getInternalScopesObject(supportedRequiredScopes),
       optionalScopes: getInternalScopesObject(supportedOptionalScopes),
       isMultichainOrigin: true,
+      sessionProperties: {},
```

Comment:
This is necessary because of [other changes introduced](https://github.com/MetaMask/core/pull/5491) to `@metamask/chain-agnostic-permission`